hat/backends/ffi/cuda/src/main/java/hat/backend/ffi/CudaBackend.java

```diff
@@ -410,11 +410,20 @@ String createPTX(KernelCallGraph kernelCallGraph, NDRange ndRange, Object... arg
         builder.ptxHeader(major, minor, target, addressSize);
         out.append(builder.getTextAndReset());
 
-        for (KernelCallGraph.KernelReachableResolvedMethodCall k : kernelCallGraph.kernelReachableResolvedStream().toList()) {
-            FuncOpWrapper calledFunc = new FuncOpWrapper(kernelCallGraph.computeContext.accelerator.lookup,k.funcOpWrapper().op());
-            FuncOpWrapper loweredFunc = calledFunc.lower();
-            loweredFunc = transformPTXPtrs(loweredFunc, argsMap, usedMathFns);
-            invokedMethods.append(createFunction(new PTXHATKernelBuilder(addressSize).nl().nl(), loweredFunc, false));
+        if (Boolean.getBoolean("moduleOp")) {
```

Comment:
Lets temporarily write out the 'mode' ("using moduleOp" or "not using moduleOp") to stdout. While we test this code.